src/bindings/js/node/CMakeLists.txt

```diff
@@ -18,14 +18,15 @@ if(WIN32)
 
     set(CMAKE_JS_NODELIB_DEF ${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/node-lib.def)
     set(CMAKE_JS_NODELIB_TARGET ${CMAKE_JS_LIB})
+    set(CMAKE_DELAY_LIB delayimp.lib)
```

Comment:
typically, `CMAKE_` prefix is used for standard cmake variables. Custom one should be named differently